WcaOnRails/app/models/round.rb

```diff
@@ -4,6 +4,10 @@ class Round < ApplicationRecord
   has_one :event, through: :competition_event
   belongs_to :format
 
+  serialize :time_limit, TimeLimit
```

Comment:
It's also very possible I'm suffering from "i just learned about this shiny new tool, and I want to use it" syndrome.

Just to be clear, I think the alternative would be to create the following columns:

- `time_limit_centiseconds`
- `time_limit_cumulative`
- `cutoff_type`
- `cutoff_ranking`
- `cutoff_percentile`
- `cutoff_attempt_value`
- `combined_round_number_of_attempts`
- `combined_round_cutoff_type`
- `combined_round_cutoff_ranking`
- `combined_round_cutoff_percentile`
- `combined_round_cutoff_attempt_value`

Perhaps @jonatanklosko has an opinion here =)